### PR TITLE
[CDAP-9253] Add tooltip to custom condition option in filter directive

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Fixes styling for `ColumnActionsDropdown` to be namespaced to avoid affecting popovers added under it
- Adds tooltip to custom condition option in filter directive

JIRA: https://issues.cask.co/browse/CDAP-9253

#### Note:
The tooltip doesn't appear on hover as it gets tricky to go inside the tooltip when its open. Tooltip is shown on click instead of hover.

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrep/Directives/Filter/index.js [hunk 1]
```diff
@@ -203,12 +206,45 @@ export default class FilterDirective extends Component {
...
+    });
+  }
+
   renderCustomFilter() {
     if (this.state.selectedCondition.substr(0, 6) !== 'CUSTOM') { return null; }
 
     return (
       <div>
-        <br />
+        <div  className="custom-condition-container">
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/DataPrep/Directives/Filter/index.js]: nit: extra space after `div`

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 2]
```diff
@@ -563,6 +563,10 @@ features:
           TEXTEXACTLY: value is
           TEXTREGEX: value matches regex
           TEXTSTARTSWITH: value starts with
+        customconditiontooltiptitle: JEXL expressions
+        customconditiontooltip: "A custom condition can be defined using JEXL expressions. For more details regarding JEXL, please refer to.  "
+        customconditiontooltiplink: JEXL Expressions
+        customconditionlabel: 'equals to: '
```
Comment (tonybach) [inline cdap-ui/app/cdap/text/text-en.yaml]: Sort these alphabetically?

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 3]
```diff
@@ -563,6 +563,10 @@ features:
           TEXTEXACTLY: value is
           TEXTREGEX: value matches regex
           TEXTSTARTSWITH: value starts with
+        customconditiontooltiptitle: JEXL expressions
+        customconditiontooltip: "A custom condition can be defined using JEXL expressions. For more details regarding JEXL, please refer to.  "
```
Comment (tonybach) [inline cdap-ui/app/cdap/text/text-en.yaml]: Do we want the period at the end?

**Review by tonybach — COMMENTED** [review]
Minor comments

**Comment by tonybach** [comment]
Code LGTM 👍

**Comment by leacd** [comment]
LGTM

**Pull request merged**